ucx.spec.in

```diff
@@ -19,7 +19,7 @@ Summary: Unified Communication X
 Version: %{version}
 Release: %{rel}
 
-License: Proprietary
+License: BSDv3
```

Comment:
Since you change the file, please change the name from openucx.org -> www.openucx.org